src/coreComponents/physicsSolvers/multiphysics/poromechanicsKernels/MultiphasePoromechanics_impl.hpp

```diff
@@ -161,16 +161,13 @@ smallStrainUpdate( localIndex const k,
                                                        dSolidDensity_dPressure );
 
   // Step 2: compute the body force
-  if( m_gravityAcceleration > 0.0 )
```

Comment:
Why do we always compute the body force even when the gravity acceleration is zero?

Reply:
I changed it in response to this [comment](https://github.com/GEOSX/GEOS/pull/2196#discussion_r1120972549) from Thomas. If we want to keep this type of optimization, it should be with `if( !isZero( m_gravityAcceleration ) )`.

I just removed these checks, because this type of optimization does not optimize much. It is also error-prone if we have a wrong check (like here) or someone removes the check in one place (call to `computeBodyForce`) and not the other (in `assembleMomentumResidual`). The last reason is that in the flow kernels, we just assemble the gravity terms all the time, so the code will be more uniform like this.